### ISSUE TITLE
minify makes mermaid subgraphs and diamonds black

### ISSUE THREAD

**Issue by pocc** [tdewolff/minify#260]
May be related to #256 and #19 

[Mermaid](https://mermaidjs.github.io/) is a diagram creation tool that can save as svg.

Mermaid subgraphs and diamonds come out black. Source mermaid:

```mermaid
graph TD
subgraph one
A[Christmas] -->|Get money| B(Go shopping)
B --> C{Let me think}
C -->|One| D[Laptop]
C -->|Two| E[iPhone]
C -->|Three| F[fa:fa-car Car]
end
```

Source and svgs included in [mermaid.zip](https://github.com/tdewolff/minify/files/3343356/mermaid.zip)

Separately, is there some way to make minify ignore an HTML div or file?

**Comment by tdewolff** [comment]
The problem is that the CSS is referencing the rectangle and giving it colors, but the minifier converted it into a `<path>` object. I think the SVG minifier should not convert rect/polypath to path, at least not when it is referenced by CSS.

**Comment by pocc** [comment]
I figured it had something to do with path conversions. Looks like adding a conditional testing for related css could be added to the elseifs: https://github.com/tdewolff/minify/blob/579cceff30c4a1872f57ee226fdf54d35a81753c/svg/svg.go#L118 and line 120.

**Comment by tdewolff** [comment]
Fixed in c05f24e69ab5bdc4ba8d8d96a38408ee1e7f7be8
